data-raw/standard_format_markdowns/combine_data_and_eda.Rmd

```diff
@@ -0,0 +1,251 @@
+                      preyDataProcessing::zeug_sanJoaquin_prey_data, 
+                      preyDataProcessing::montgomery_prey_data, 
+                      preyDataProcessing::guignard_prey_data, 
+                      preyDataProcessing::corline_prey_data,
+                      preyDataProcessing::zooper_prey_data) 
+
+# usethis::use_data(all_prey_density, overwrite = TRUE)
+
+```
+
+
+### Create Monthly Prey Density Dataset
+
+**Notes:**
+
+* There are two NAs associated with the month of San Joaquin data because data received was aggregated to year
+
+```{r}
+# monthly aggregation by watershed,  habitat type -------------------------
+## group by watershed, habitat type, year, month - summarize prey density (median, min, max)
+monthly_prey_density <- all_prey_density %>%
+  mutate(year = ifelse(!is.na(date), lubridate::year(date), 2016),
+         month = lubridate::month(date)) %>%
+  group_by(watershed, habitat_type, year, month) %>%
+  summarise_at(vars(prey_density), .funs = c('min', 'max', 'median', 'var'), na.rm = TRUE) %>%
+  ungroup() %>%
+  rename(min_prey_density = min,
+         max_prey_density = max,
+         median_prey_density = median,
+         var_prey_density = var) |> glimpse()
+
+# save file for use in R package
+# usethis::use_data(monthly_prey_density)
+
+```
+
+
+<!-- ## Fish Data  -->
+
+<!-- ```{r} -->
+
+<!-- load('../data/all_fish_data.rda') -->
+
+<!-- #all_fish_data <- all_fish_data %>% select(-lat, -lon) -->
+<!-- #usethis::use_data(all_fish_data, overwrite = TRUE) -->
+
+<!-- monthly_fish_data <- all_fish_data %>% -->
+<!--   mutate(year = ifelse(!is.na(date), lubridate::year(date), 2016), -->
+<!--          month = lubridate::month(date)) %>% -->
+<!--   mutate(habitat_type = case_when(watershed == "Merced" ~ "perennial instream", -->
+<!--                                   watershed != "Merced" ~ habitat_type)) %>% -->
+<!--   group_by(watershed, habitat_type, year, month) %>% -->
+<!--   summarise_at(vars(mass, fork_length), .funs = c('min', 'max', 'median'), na.rm = TRUE) %>% -->
+<!--   ungroup() |> glimpse() -->
+
+<!-- # save file for use in R package -->
+<!-- # usethis::use_data(monthly_fish_data) -->
+
+<!-- ``` -->
+
+<!-- ## Environmental Data  -->
+
+<!-- ```{r} -->
+<!-- all_enviro <- preyDataProcessing::all_enviro_data %>% glimpse -->
+
+<!-- ``` -->
+
+# Explore Variables
+
+## Authors
+
+```{r}
+
+prey_agg_by_author <- preyDataProcessing::all_prey_density %>%
+  group_by(author) %>%
+  summarise_at(vars(prey_density), .funs = c('min', 'max', 'median', 'var'), na.rm = TRUE)
+
+kable(prey_agg_by_author, digits = 1, caption = "summary statistics of prey density values across authors included in the prey data package")
+
+```
+
+## Habitat Type
+
+Habitat types included in the prey data package include: 
+
+* Agricultural canal
+* Floodplain
+* Perennial instream
+* Side channel
+
+Many habitat types were assigned based on the sampling locations provided by authors. 
```

Comment:
Please update: Habitat types were assigned based on sapling locations provided by authors where available. When no habitat types were provided by authors, we assigned habitat type based on satellite imagery. Please see (link to hab locations md) for more details.